pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v3/Transactions.java

```diff
@@ -314,4 +316,25 @@ public void getPendingAckInternalStats(@Suspended final AsyncResponse asyncRespo
             resumeAsyncResponseExceptionally(asyncResponse, ex);
         }
     }
+
+    @POST
+    @Path("/transactionCoordinator/replicas")
+    @ApiResponses(value = {
+            @ApiResponse(code = 503, message = "This Broker is not configured "
+                    + "with transactionCoordinatorEnabled=true."),
```

Comment:
Have you added this check in the implementation?
And should also add 

```
            @ApiResponse(code = 401, message = "This operation requires super-user access"),
```

Reply:
`checkTransactionCoordinatorEnabled(); ` had been aaded, and I added auth and auth test this time.